src/02_spec/07_modules/dem_uart/index.rst

```diff
@@ -0,0 +1,73 @@
+UART Device Emulation Module (DEM-UART)
+=======================================
+
+.. figure:: img/osd_dem_uart.*
+   :alt: Device Emulation Module UART
+
+The UART Device Emulation Module connects to the bus of a given CPU on one side, and to the Debug Interconnect on the other.
+Towards the CPU it behaves like a UART-16550A device, but instead of transmitting information over a UART-Interface it instead passes it to a host PC via the Debug Interconnect.
+
+.. _uart_registers:
+
+16550 UART Registers
+--------------------
+The following UART registers are implemented and accessible via the bus, the address mapping is in accordance with the UART-16550(A) standard as specified in this `Datasheet <http://caro.su/msx/ocm_de1/16550.pdf>`_.
+Currently there are no FIFOs present in the hardware, therefore writing to the ``FCR`` has no effect.
+The only bit in the ``LCR`` that has any meaning is the DLAB aka ``LCR7``, all other bits hold their written value but have no meaning.
+No Modem or DMA-Mode related features, registers or interrupts are implemented.
+Writing to a register that is not implemented has no effect, reading from such a register will always return ``0x00``.
+
+All registers are 8 bit wide.
+
+.. tabularcolumns:: |p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.40\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|
+.. flat-table:: 16550 UART Registers
+  :widths: 1 1 2 4 1
+  :header-rows: 1
+
+  * - Address
+    - Register
+    - Access Type
+    - Reset Value
+    - Description
+
+  * - ``0x00``
+    - ``RBR``
+    - Read only
+    - ``0x00``
+    - Receiver Buffer Register
+
+  * - ``0x00``
+    - ``THR``
+    - Write only
+    - ``0x00``
+    - Transmitter Holding Register
+
+  * - ``0x01``
+    - ``IER``
+    - Read/Write
+    - ``0x00``
+    - Enable(1)/Disable(0) interrupts. See `this <http://caro.su/msx/ocm_de1/16550.pdf>` for more details on each interrupt.
+
+  * - ``0x02``
+    - ``IIR``
+    - Read only
+    - ``0x01``
+    - Information which interrupt occurred
+
+  * - ``0x02``
+    - ``FCR``
+    - Write only
+    - ``0x00``
+    - Control behavior of the internal FIFOs
+
+  * - ``0x03``
+    - ``LCR``
+    - Read/Write
+    - ``0x00``
+    - Access to the DLAB
```

Comment:
You might want to have the information about this register that you mentioned in the introduction placed here. In the introduction you can stay general.